### PR TITLE
added netbird relay

### PR DESCRIPTION
**Pull request by Chaosjan44**

### CODE REVIEW

**Comment by Chaosjan44** [comment]
@schmidt-silas pls and merge ☕

File: netbird/build_files/setup.env [hunk 1]
```diff
@@ -19,4 +19,8 @@ NETBIRD_MGMT_API_PORT=443
 NETBIRD_SIGNAL_PORT=443
 TURN_MIN_PORT=49152
 TURN_MAX_PORT=60999
-NETBIRD_DISABLE_LETSENCRYPT=true
\ No newline at end of file
+NETBIRD_DISABLE_LETSENCRYPT=true
+
+## Relay
+NETBIRD_RELAY_DOMAIN=your.netbird.domain
```
Comment (schmidt-silas) [inline netbird/build_files/setup.env:25]: $NETBIRD_DOMAIN

**Review by schmidt-silas — CHANGES_REQUESTED** [review]
file name and variable

**Comment by Chaosjan44** [comment]
changed it

**Review by schmidt-silas — APPROVED** [review]
should work, trust me bro :D

**Comment by schmidt-silas** [comment]
It didn't xD

**Pull request merged**